<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginsDialog</class>
 <widget class="QDialog" name="PluginsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Plugins</string>
  </property>
  <property name="windowIcon">
   <iconset resource="photosdd.qrc">
    <normaloff>:/icon/photosdd</normaloff>:/icon/photosdd</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="upButton">
     <property name="text">
      <string>move up</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="downButton">
     <property name="text">
      <string>move down</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="photosdd.qrc"/>
 </resources>
 <connections/>
</ui>
